packages/mask/src/social-network-adaptor/facebook.com/utils/selector.ts

```diff
@@ -0,0 +1,63 @@
+import { LiveSelector } from '@dimensiondev/holoflows-kit'
+
+type E = HTMLElement
+
+const querySelector = <T extends E, SingleMode extends boolean = true>(selector: string, singleMode = true) => {
+    const ls = new LiveSelector<T, SingleMode>().querySelector<T>(selector)
+    return (singleMode ? ls.enableSingleMode() : ls) as LiveSelector<T, SingleMode>
+}
+
+const querySelectorAll = <T extends E>(selector: string) => {
+    return new LiveSelector().querySelectorAll<T>(selector)
+}
+
+export const searchUserIdOnMobileSelector: () => LiveSelector<E, true> = () =>
+    querySelector<HTMLMetaElement>('div[data-sigil$="profile"] a')
+
+export const searchAvatarSelector: () => LiveSelector<E, true> = () => querySelector<E>('[href="/me/"] image')
```

Comment:
you sure they're using `image` tag?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/image

Reply:
Yes